test/image/mocks/axes_breaks-heatmap1d.json

```diff
@@ -0,0 +1,111 @@
+{
+    "data": [
+        {
+            "type": "heatmap",
+            "x": [
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10",
+                "2020-01-02 17:00",
+                "2020-01-02 17:10",
+                "2020-01-02 17:20",
+                "2020-01-02 17:30",
+                "2020-01-02 17:40",
+                "2020-01-02 17:50",
+                "2020-01-02 18:00",
+                "2020-01-02 18:10"
```

Comment:
These tests include some removed data, which is good - but can we also include some data on the other side of the break, ie the next morning?

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4821/commits/e2cf395aeec7ea1c2950a7579d157671f1c85fdb.